src/arrows.jl

```diff
@@ -117,6 +144,31 @@ function arrow(centerpos::Point, radius, startangle, endangle;
     grestore()
 end
 
+"""
+    arrow(left::Point, C1::Point, C2::Point, right::Point, action=:fill;
+        linewidth=1.0, headlength=10, headangle=pi/8, leftarrow=false, rightarrow=true)
+
+Draw an Bezier curved arrow, from `left` to `right`, arrow head can be added by tweaking
+flags `leftarrow` and `rightarrow`.
+"""
+function arrow(left::Point, C1::Point, C2::Point, right::Point, action=:fill;
+    linewidth=1.0, headlength=10, headangle=pi/8, leftarrow=false, rightarrow=true)
+    gsave()
+    setline(linewidth)
+
+    move(left)
+    curve(C1, C2, right)
+    do_action(:stroke)
+
+    v = right - C2
+    right_shaftangle = atan(v.y / v.x)
+    v = left - C1
+    left_shaftangle = atan(v.y / v.x)
+    rightarrow && arrowhead(right, action, shaftangle=pi + right_shaftangle)
+    leftarrow && arrowhead(left, action, shaftangle=left_shaftangle)
+    grestore()
+end
+
```

Comment:
A couple of suggestions.... 

1: Instead of left/right, you could have 'start'/'finish'. So if you have vertical arrows, it still makes sense...

 2: Instead of `atan`, you can use:

```
left_shaftangle  = slope(left, C1)
right_shaftangle = slope(C2, right)
```
which will produce more expected results in some cases. (Or do you want the arrowheads to always point in the same direction?)

![arrow](https://user-images.githubusercontent.com/52289/54526895-1c365b80-4970-11e9-8101-543afed16499.png)

Reply:
sounds good, I think you are right, I intended to have the blue one, those two arrows should have opposite direction.